templates/password.php

```diff
@@ -33,8 +33,7 @@
 <form id="password_policy" method="post">
 	<fieldset>
 		<h1 class="warning">
-			<div><?php p($l->t('Your password has expired.')); ?></div>
```

Comment:
we need to keep this message when the password has actually expired

it seems this PR completely removes the existing messages ?